composer/content.md

```diff
@@ -23,7 +23,8 @@ $ docker run --rm --interactive --tty \
 You can bind mount the Composer home directory from your host to the container to enable a persistent cache or share global configuration:
 
 ```console
-$ docker run --rm --interactive --tty \
+$ COMPOSER_HOME=$HOME/.composer \
+  docker run --rm --interactive --tty \
   --volume $PWD:/app \
   --volume $COMPOSER_HOME:/tmp \
```

Comment:
This PR looks great except this bit -- I don't think this does what you want/intend:

```console
$ FOO=bar echo $FOO

$ FOO=bar; echo $FOO
bar
```

Perhaps you really intended for something like `--volume ${COMPOSER_HOME:-$HOME/.composer}:/tmp`?

Reply:
You are right